tests/pytorch/mock_sparse2/test_reduction.py

```diff
@@ -0,0 +1,131 @@
+import operator
```

Comment:
Ideally the tests should cover the examples.

Reply:
I used `doctest` module to cover all docstrings automatically.  We actually should do it for other modules as well.